src/common/eeprom.h

```diff
@@ -49,14 +76,22 @@ extern void eeprom_set_var(uint8_t id, variant8_t var);
 // fill range 0x0000..0x0800 with 0xff
 extern void eeprom_clear(void);
 
-// fill dest parameter with hostname (max 20 chars)
-extern void eeprom_get_hostname(char *dest);
+//
+extern int eeprom_load_bin_from_usb(const char* fn);
```

Comment:
eeprom serialization is an unnecessary dependence here
this can be done in a separate h/c/cpp file to minimize the interface of the eeprom itself

Reply:
i will move serialization functions to separate source

Reply:
commit 3c634b7